src/main/java/org/tdl/vireo/util/FileIOUtility.java

```diff
@@ -108,5 +115,16 @@ private Path processRelativePath(String relativePath) throws IOException {
         }
         return path;
     }
+    
+    public File getFileFromResource(String resourcePath) throws IOException {
```

Comment:
Probably don't need this in this utility. Not sure this file IO utility is a good utility in the first place.